features/upgrade/build/build-upgrade.feature

```diff
@@ -0,0 +1,41 @@
+Feature: build related upgrade check
+  # @author wewang@redhat.com
+  @upgrade-prepare
+  @users=upuser1,upuser2
+  Scenario: basic verification for upgrade build test
+    Given I switch to the first user
+    When I run the :new_project client command with:
+      | project_name | build-upgrade |
+    Then the step should succeed
+    When I run the :new_app client command with:
+      | app_repo | openshift/ruby~https://github.com/openshift/ruby-ex |
+    Then the step should succeed
+    When I run the :new_app client command with:
+      | app_repo | openshift/ruby~https://github.com/openshift/ruby-hello-world |
+      | strategy | docker                                                       |
+    Then the step should succeed
+    Given I use the "build-upgrade" project
```

Comment:
Looks like we do not need this step.

Reply:
It will report "no projects in cache" when there's no the step

Reply:
Ack